templates/landscape/install.html

```diff
@@ -424,10 +220,16 @@ <h3 class="p-heading--2">Configure SSL</h3>
                     <pre class="p-code-snippet__block is-wrapped"><code>sudo snap install certbot --classic</code></pre>
                   </div>
                 </li>
+                <li class="p-list__item">
+                  <p class="u-no-padding--top">Set your email address in a variables</p>
```

Comment:
Please revise this to use a singular form of the word "variable": `Set your email address in a variable` (remove trailing `s`)